src/Mod/Draft/DraftSnap.py

```diff
@@ -1480,11 +1505,12 @@ def addHoldPoint(self):
                 self.holdTracker.addCoords(self.spoint)
                 self.holdTracker.on()
             self.holdPoints.append(self.spoint)
+        
 
 if not hasattr(FreeCADGui,"Snapper"):
     FreeCADGui.Snapper = Snapper()
 if not hasattr(FreeCAD,"DraftWorkingPlane"):
-    import WorkingPlane, Draft_rc
+    import WorkingPlane
     FreeCAD.DraftWorkingPlane = WorkingPlane.plane()
     #print(FreeCAD.DraftWorkingPlane)
     FreeCADGui.addIconPath(":/icons")
```

Comment:
If I'm not mistaken, this initialization code was removed in my branches.

It is messy to initialize inside of a module because it is doing things behind the scenes without us knowing. It is better to define the classes and methods in the module `DraftSnap`, then we can import it, and initialize it in  `DraftTools`. See #3091.
```py
import DraftSnap

Gui.Snapper = DraftSnap.Snapper()
```